PSFScriptSample/CPPPSFScriptSample/CPPPSFScriptSample.vcxproj

```diff
@@ -0,0 +1,175 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
>Project [](start = 1, length = 7)

Consider looking at PsfLauncher.vcxproj to keep the project configurations common. This project breaks the 10.0.14393.0 target of the solution.